packages/plugins/i18n/admin/src/components/CMEditViewInjectedComponents/CMEditViewCopyLocale/index.js

```diff
@@ -59,12 +59,15 @@ const Content = ({ appLocales, currentLocale, localizations, readPermissions })
 
     const requestURL = `/content-manager/collection-types/${slug}/${value}`;
 
+    setIsLoading(true);
     try {
-      setIsLoading(true);
-
       const { data: response } = await axiosInstance.get(requestURL);
 
       const cleanedData = cleanData(response, allLayoutData, localizations);
+      ['createdBy', 'updatedBy', 'publishedAt', 'id', '_id', 'createdAt'].forEach(key => {
```

Comment:
This field were removed on purpose as we creating a new entry they should be removed.

I don't think it is the correct fix for #12002

Reply:
That's exactly why I put `if (!initialData[key]) return;` statement in code.
When you create a new entry, it neither has nor needs those fields. But when you "edit" an existing entry you should not omit them.

Reply:
`'_id'` there is no reason for this to exist anymore, it was a mongodb thing that shouldn't exist anymore

Reply:
@derrickmehaffy 
Yeah, I just copied them from `cleanData`, should I remove both of them?
https://github.com/strapi/strapi/blob/48893ae3fc951b618fd8c4fdc6970e623d2c92db/packages/plugins/i18n/admin/src/components/CMEditViewInjectedComponents/CMEditViewCopyLocale/utils/cleanData.js#L16-L24

Reply:
@iicdii my bad :see_no_evil: you were right I misunderstood the situation the bug occurs when editing an entry...

Your code is correct we just need to remove the `_id` and then we can merge it.


Thank you for your contribution!

Reply:
Thanks!
I just remove them and wait for passing the tests..